app/templates/create-masterclass/content/create-new.html

```diff
@@ -0,0 +1,80 @@
+{%  extends "base.html" %}
+
+{% block content %}
+<div class="govuk-width-container">
+  <a href="/choose-content/new-or-existing" class="govuk-back-link">Back</a>
+
+  <main class="govuk-main-wrapper " id="main-content" role="main">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl">Create new masterclass content</h1>
+        <form action="/choose-content/create-new" method="post" novalidate>
+            <div class="govuk-form-group">
+                <label class="govuk-label" for="event-name">
+                    Name
+                </label>
+                <span id="more-detail-hint" class="govuk-hint">
+                    Summarise as best you can what this masterclass will cover
+                </span>
+                <input class="govuk-input" id="masterclass-name" name="masterclass-name" type="text">
+              </div>
+            <div class="govuk-form-group">
+                <label class="govuk-label" for="more-detail">
+                    Description
+                </label>
+                <span id="more-detail-hint" class="govuk-hint">
+                    Describe what someone attending the masterclass would learn from it
+                </span>
+                <textarea class="govuk-textarea" id="masterclass-description" name="masterclass-description" rows="5" aria-describedby="masterclass-description"></textarea>
+                <div class="govuk-radios">
+                    <label class="govuk-label" for="sort">
+                      Which DDaT 
```

Comment:
Should be: Which DDaT job family is your masterclass related to?

Reply:
Could extract DDaT job family radio buttons into separate template to make more DRY.

Reply:
Made into a partial template.